### PR TITLE
Dropdown component update

### PR DESCRIPTION
**Pull request by Malinicz**
- update Dropdown with 2 new props (fullWidth, colorMode)
- fix issue with run-sequence in watch mode

image

### CODE REVIEW

File: src/components/dropdowns/Dropdown.jsx [hunk 1]
```diff
@@ -14,13 +14,23 @@ type PropsType = $ReadOnly<{
   name: string,
   links: Array<LinksType>,
   initiallyOpened?: boolean,
+  colorMode?: 'dark' | 'light',
```
Comment (bartoszlorek) [inline src/components/dropdowns/Dropdown.jsx]: I'd keep what we have for selects `color?: 'default' | 'white'`

File: src/components/dropdowns/Dropdown.jsx [hunk 2]
```diff
@@ -41,7 +51,11 @@ const Dropdown = ({name, links, initiallyOpened}: PropsType) => {
 
   return (
     <div
-      className={classNames('sg-dropdown', {'sg-dropdown--opened': open})}
+      className={classNames('sg-dropdown', {
+        'sg-dropdown--opened': open,
+        'sg-dropdown--light': lightMode,
```
Comment (bartoszlorek) [inline src/components/dropdowns/Dropdown.jsx]: and then `sg-dropdown--white`

File: src/components/dropdowns/Dropdown.jsx (hunk 1 above)
Comment (Malinicz) [reply src/components/dropdowns/Dropdown.jsx]: sure, fixed - I guess in this case it makes sense to keep it consistent with Select component, however I'm not a huge fan of naming things with color names as if in the future we change the color to e.g. light gray, giving it prop color="white' could be slightly confusing :) @bartoszlorek

File: src/components/dropdowns/Dropdown.jsx (hunk 2 above)
Comment (Malinicz) [reply src/components/dropdowns/Dropdown.jsx]: fixed

**Pull request merged**